docs/en/how-to/administrators-receivers.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Administrators or Receivers Appointed
+---
+
+If Administrators or Receivers are appointed then realistically your securities are probably worthless.
```

Comment:
Can you elaborate a little bit on this. I have no clue what this could signify (Administrators or Receivers). Thanks

Reply:
Also, I don't get a message when you send a Pull Request. Is there a setting that I should activate?

Reply:
Maybe this is better but the file should be renamed.

Reply:
I'd expect you would get a notice for all activity on the repo. In the repo you could add a `Watch` on `All activity`.